css/jquery.uls.lcd.css

```diff
@@ -119,7 +119,7 @@
 }
 
 .uls-no-results-found-title {
-	font-size: 16px;
+	font-size: 1.2em;
```

Comment:
Same as above.

Reply:
Weirdly `.uls-no-found-more` below has always had `font-size: 0.9em`.

Reply:
While at it, the two `line-height` values should loose `em`. For below the better choice is
`line-height: 1.6`

No value unit here takes the current font-size and multiplies it. So no matter if you choose `px`/`rem`/`em` font-size, you'll always end up with a good multiplier value.